mvpa2/support/nibabel/surf.py

```diff
@@ -1975,15 +1954,15 @@ def generate_cube():
 
     # manually set topology
     trias = [(0, 1, 3), (0, 3, 2), (1, 0, 5), (5, 0, 4),
-           (3, 1, 5), (3, 5, 7), (3, 7, 6), (3, 6, 2),
-           (2, 6, 0), (0, 6, 4), (5, 4, 6), (5, 6, 7)]
-
+             (3, 1, 5), (3, 5, 7), (3, 7, 6), (3, 6, 2),
+             (2, 6, 0), (0, 6, 4), (5, 4, 6), (5, 6, 7)]
 
     fs = np.asarray(trias, dtype=np.int)
 
     return Surface(vs, fs)
 
 
+
```

Comment:
the void multiplies.

1 or 2 spaces is more than enough (pycharm suggests 1 between methods iirc and 2 between classes and functions)